CHANGELOG.md

```diff
@@ -9,9 +9,21 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 [Unreleased][Unreleased_log]
 ------------
+
 ### Fixed
 - Fix #2607 so that libmbedcrypto now includes mbedtls_hkdf().
```

Comment:
Is the plan for master to go into 0.8.2 or are we backporting only the subset of changes listed below?

Reply:
We are only backporting a small subset of changes(listed in the 0.8.2 section)